tests/unit/test_addtoken.py

```diff
@@ -0,0 +1,10 @@
+from unittest import mock, TestCase
+from add_token import print_short
+
+class test_at(TestCase):
+    @mock.patch('builtins.print')
+    @mock.patch('add_token.get_network_info', return_value=['10.23.53.54', ['10.23.53.54'], str(32)])
```

Comment:
let's be consistent and use `'32'` as well, instead of `str(32)`

Reply:
Got it.